### PR TITLE
Contributing + Variants docs

### PR DESCRIPTION
**Pull request by gillkyle**
### Added
- small updates to give additional examples for variants
- additional links to styled system (like explanations)
- and more steps in the contribution guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,12 +15,22 @@ so please be patient if a particular feature takes longer to review than others.
...
 
 ```sh
 yarn
 ```
 
+After yarn has linked packages and installed dependences in the repo you can run the docs or an
+example site in the workspace with this command:
+
+```sh
+yarn start <name-of-package>
```
Comment (jxnblk) [inline CONTRIBUTING.md]: TIL: didn’t know about this shortcut – yarn is so unguessable

File: packages/docs/src/pages/theme-spec.mdx [hunk 2]
```diff
@@ -173,10 +173,17 @@ This does not mean that these values are in any way tied to specific heading lev
 ## Variants
 
 Theme UI currently does _not_ have standards for defining combinations of typographic styles, but _variants_ can be used for situations where it is useful to reuse combinations of these primitive values.
+Another common practice is to have a `buttons` object with variants like primary and secondary:
 
 ```js
 // example typographic style variants
 {
+  colos: {
```
Comment (jxnblk) [inline packages/docs/src/pages/theme-spec.mdx]:
```suggestion
  colors: {
```

File: packages/docs/src/pages/theme-spec.mdx [hunk 3]
```diff
@@ -203,27 +210,45 @@ Theme UI currently does _not_ have standards for defining combinations of typogr
       letterSpacing: '0.1em',
     },
   },
+  // variants for buttons
+  buttons: {
+    primary: {
+      color: colors.white, // you can reference other values defined in the theme
```
Comment (jxnblk) [inline packages/docs/src/pages/theme-spec.mdx]: This *should* work

```suggestion
      color: 'white'  // you can reference other values defined in the theme
```

**Comment by jxnblk** [comment]
Thanks!

**Comment by gillkyle** [comment]
Woops sorry, saw you already applied those suggestions in another PR, thanks!

**Comment by jxnblk** [comment]
I *think* I forgot to apply the suggestions here before merging, but it's all good. Mostly wanted to make sure you saw them for future reference :)

**Pull request merged**